### PR TITLE
Better colors for dark mode select and tagsinput components

### LINKED ISSUE #1547: Incorrect font colour on share file modal (dark mode)
**Description**
- It's possible to type in the share drop down input in order to quickly find an existing shared folder but the text is too dark to see clearly

**Steps to Reproduce**
1) Login to files with dark theme enabled
2) Attempt to share a file
3) Type in the drop down selector to quickly locate the folder you want to share too

**Screenshots**

Screen Shot 2021-09-17 at 11 16 26 AM

**Desktop:**
 - OS: macOS
 - Browser: Chrome
 - Version : 93.0.4577.63

### PR DESCRIPTION
**Pull request by Tbaut**
closes #1547 

https://user-images.githubusercontent.com/33178835/134155904-0ccfb5f7-0772-46eb-9309-31bf30fc9d80.mp4

### CODE REVIEW

**Review by asnaith — APPROVED** [review]
Nice, looking much better now 😎

**Comment by sweetpea22** [comment]
you're an artiste THANK YOU!! ❤️

File: packages/common-components/src/SelectInput/SelectInput.tsx [hunk 1]
```diff
@@ -121,6 +116,7 @@ const SelectInput: React.FC<ISelectInputProps> = ({
       )
     })
   })
+
   selectOverides.container = (provided, state) => ({
```
Comment (FSM1) [inline packages/common-components/src/SelectInput/SelectInput.tsx]:
```suggestion
  selectOverrides.container = (provided, state) => ({
```

**Review by FSM1 — APPROVED** [review]
just a minor spelling error that was inherited

File: packages/common-components/src/SelectInput/SelectInput.tsx (hunk 1 above)
Comment (Tbaut) [reply packages/common-components/src/SelectInput/SelectInput.tsx]: ah yup, fixed :)

**Pull request merged**